File: Addition Game.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
"""
Created on Fri May 20 09:20:00 2022

@author: michaelconnors
"""

import random

print("Welcome to the math game!")
print("Type in the answer to the question.")

correct = 0
incorrect = 0
total = 0

while True:

    num1 = random.randint(1, 10)
    num2 = random.randint(1, 10)
    ans = int(num1) + int(num2)
    print(f"  {num1}\n+ {num2}")
    print("------")
    player_ans = int(input())

    if player_ans == ans:
            print("Correct!")
            correct = correct + 1
            total = total + 1

                
    elif player_ans != ans:
            print("Sorry, that is incorrect.")
            incorrect = incorrect + 1
            total = total +1

            
    play_again = input("Play again? y or n")
    if play_again != "y":
        break

print("Great job!")
print(f"Total rounds played: {total}")
print(f"Total correct: {correct}")
print(f"Total incorrect: {incorrect}")
print("Thanks for playing!")
        
        
    

File: Addition_Game.java
import java.util.Random;
import java.util.Scanner;

public class Addition_Game {

	public static void main(String[] args) {
		
		Random rand = new Random(System.currentTimeMillis());
		Scanner scan = new Scanner(System.in);
		
		int correct = 0;
		int incorrect = 0;
		int total = 0;
		String play_again;
		
		System.out.println("Welcome to the math game!");
		System.out.println("Type in the answer to the question.");
		
		do {
			int num1 = rand.nextInt(11);
			int num2 = rand.nextInt(11);
			int num3 = num1 + num2;
			
			System.out.println(num1 + " + " + num2 + " = ");
			int ans = scan.nextInt();
			if(ans == num3) {
				System.out.println("Correct!");
				correct ++;
				total++;
			} else {
				System.out.println("Incorrect");
				incorrect++;
				total++;
			}
			System.out.println("Do you want to play again? y or n");
			play_again = scan.next();
			
		}while(play_again.equals("y"));
		
		System.out.println("Great job!");
		System.out.println("Total correct: " + correct);
		System.out.println("Total incorrect: " + incorrect);
		System.out.println("Total rounds: " + total);
	}
}
	
		